components/skus/browser/rs/wasm/Cargo.lock

```diff
@@ -1074,6 +1115,6 @@ checksum = "712e227841d057c1ee1cd2fb22fa7e5a5461ae8e48fa2ca79ec42cfc1931183f"
 
 [[package]]
 name = "zeroize"
-version = "1.4.3"
+version = "1.3.0"
```

Comment:
Downgrade to match challenge-bypass-ristretto and curve25519-dalek because of api drift in 1.4. Ok.